addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -265,7 +274,15 @@ import { sprintf } from "@web/core/utils/strings";
 
             self.$el.find('#s_website_form_result, #o_website_form_result').empty(); // !compatibility
             if (!self.check_error_fields({})) {
-                self.update_status('error', _t("Please fill in the form correctly."));
+                if (self.fileInputError) {
+                    const errorMessage = self.fileInputError.type === 'number'
+                        ? _.str.sprintf(_t("Please fill in the form correctly. You uploaded too many files. (Maximum %s files)"), self.fileInputError.limit)
+                        : _.str.sprintf(_t("Please fill in the form correctly. The file \"%s\" is too big. (Maximum %s MB)"), self.fileInputError.fileName, self.fileInputError.limit);
```

Comment:
Note that you could go with 
```suggestion
                        ? _.str.sprintf(_t(
                            "Please fill in the form correctly. You uploaded too many files. (Maximum %s files)"
                        ), self.fileInputError.limit)
                        : _.str.sprintf(_t(
                            "Please fill in the form correctly. The file \"%s\" is too big. (Maximum %s MB)"
                        ), self.fileInputError.fileName, self.fileInputError.limit);(function () {
```
It will remove a good chunk of the line, making it overflow just a bit over the 100 char which is reasonnable.
(Just passing by :wave: , saw the mail)